specification/analysisservices/resource-manager/Microsoft.AnalysisServices/2017-07-14/analysisservices.json

```diff
@@ -660,7 +660,7 @@
           ],
           "x-ms-enum": {
             "name": "Status",
-            "modelAsString": true
+            "modelAsString": false
```

Comment:
no, this was my fault when setting the parameter as true.
We do not have sdk released for the version yet.